subsystem/host_test.go

```diff
@@ -39,6 +39,41 @@ var _ = Describe("Host tests", func() {
 		infraEnvID = registerInfraEnv(&clusterID, models.ImageTypeMinimalIso).ID
 	})
 
+	It("Should reject hostname if it is forbidden", func() {
+		host := &registerHost(*infraEnvID).Host
+		host = getHostV2(*infraEnvID, *host.ID)
+		Expect(host).NotTo(BeNil())
+
+		for _, hostname := range hostutil.ForbiddenHostnames {
```

Comment:
Tests should preferably not make use of non-test constants, as that kinda in a way defeats the purpose of a test

Reply:
Yes, fair point, I will change this